app.js

```diff
@@ -14,87 +14,65 @@
 // along with Pa11y Webservice.  If not, see <http://www.gnu.org/licenses/>.
 'use strict';
 
-const async = require('async');
 const Hapi = require('@hapi/hapi');
 const MongoClient = require('mongodb').MongoClient;
 
-module.exports = initApp;
-
 // Initialise the application
-function initApp(config, callback) {
+module.exports = async (config, callback) => {
```

Comment:
If this function error causes an error it will break calling code that doesn't add a `.catch` on the returned promise. I think we should not directly export this function, instead rename it back to `initApp`. And then export a function that ensures that the promise rejection would be `.catch`'ed